### ISSUE TITLE
Liqudity Lock Checker

### ISSUE THREAD

**Issue by Eloonpooch** [psdlt/roboape#9]
First of all Thanks Dev for creating this Awwsome Project . It would be really great if there can be a function of checking the liqudity Lock .

**Comment by QCharger** [comment]
Hi there yes that would be great as well. Thanks dev for your effort to take out time to develop this for us

**Comment by TJCares1966** [comment]
Yes, that would be awesome.  Thanks for making this and keeping it free!
